core/tests/test_apps.management.recovery_device.py

```diff
@@ -140,6 +142,48 @@ def test_exceptions(self):
             secret, share = process_slip39(words)
             self.assertIsNone(secret)
 
+    @mock_storage
+    def test_check_word_validity(self):
+        storage.recovery.set_in_progress(True)
+
+        # We claim to know the backup type, but nothing is stored. That is an invalid state.
+        with self.assertRaises(RuntimeError):
+            check(BackupType.Slip39_Advanced, ["ocean"])
+
+        # if backup type is not set we can not do any checks
+        self.assertIsNone(check(None, ["ocean"]))
```

Comment:
i think `assertIsNone` is not appropriate here, and it would be better to have a bare `check()` call? what we actually want is something like "assertSucceeds". That's the default assumption though. We could be extra explicit and write `assertNotRaises` method, but it seems to bring no benefit.

Reply:
I agree, this is an oversight from the previous version without exceptions.

Reply:
Fixed.